testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/jpa/hibernate/entitytest/EntityTestCase.java

```diff
@@ -70,6 +72,11 @@ protected <T> T rawLookup(String name, Class<T> interfaceType) throws NamingExce
         return interfaceType.cast(iniCtx.lookup(name));
     }
 
+    // This test needs to be recompiled against Hibernate ORM 6 (WFLY-16178) in order to pass.
+    @BeforeClass
+    public static void beforeClass() {
+        assumeTrue(System.getProperty("ts.ee9") == null);
```

Comment:
You need to check for ts.bootable.ee9 as well as that enables the profile used in jobs like https://ci.wildfly.org/viewType.html?buildTypeId=WF_MainNightlyJobs_WildFlyPreviewBootableJarLinuxJdk11

Reply:
Thanks Brian, added!

Reply:
Somehow I lost these changes, making them again via `assumeTrue(System.getProperty("ts.ee9") == null && System.getProperty("ts.bootable.ee9") == null)`